numerapi/numerapi.py

```diff
@@ -155,30 +155,35 @@ def get_latest_data_url(self, data_type: str, extension: str = "csv") -> str:
 
         return url
 
-    # Convenience functions for get_latest_data_url()
-    def get_latest_live_data_url(self, extension: str = "csv") -> str:
-        return self.get_latest_data_url("live", extension)
+    def download_latest_data(self, data_type: str, extension: str, dest_path: str):
+        url = self.get_latest_data_url(data_type, extension)
+        utils.download_file(url, dest_path, self.show_progress_bars)
 
-    def get_latest_training_data_url(self, extension: str = "csv") -> str:
-        return self.get_latest_data_url("training", extension)
+    # Convenience functions for download_latest_data()
+    # The extension argument must be one of: "csv", "csv.xz", or "parquet"
```

Comment:
It might make sense to delete these "convenience" functions and only keep `download_latest_data()`

Reply:
I agree

Reply:
Done!